Sources/StylableScrollView/Extensions/Modifiers.swift

```diff
@@ -85,18 +86,21 @@ public extension ScrollView {
     ///   - title: The title that appears on top of the header.
     ///   - navBar: The contents of the navigationBar that appears when scrolling.
     ///
-    @ViewBuilder @inlinable func stretchableHeader<Title, NavBar>(
+    @ViewBuilder @inlinable func stretchableHeader<Title, NavBar, TrailingElements, LeadingElements>(
         header: Image,
         title: () -> Title,
-        navBar: () -> NavBar
-    ) -> some View where Title: View, NavBar: View {
+        navBar: () -> NavBar,
+        trailingElements: @escaping (NavigationBarProxy) -> TrailingElements,
+        leadingElements: @escaping (NavigationBarProxy) -> LeadingElements
```

Comment:
Shouldn't this be the other way around, from leading to trailing?

Reply:
Fixed by f36a2b3.